### ISSUE TITLE
Reference material spec for text scaling when available in AppBar

### ISSUE THREAD

**Issue by perclasson** [flutter/flutter#58769]
Follow up from https://github.com/flutter/flutter/issues/58093

When the material spec for accessibility text scaling is available add reference to it in docs, to educate and explain the text scale factor cap of `1.34` for titles.

**Comment by LHaanappel** [comment]
Can someone explain to me where the cap of 1.34 comes from and how it is in line with the WCAG guidelines?
Our app fails criterium 1.4.4 (200 percent text scaling), so we disabled the cap of 1.34. Are titles excluded from this criterium?

In general, the non-lineair font scaling confuses our team how it is in line with this criterium.

**Comment by timmyrosen** [comment]
@LHaanappel How did you manage to disable 1.34 cap for AppBar?

**Comment by chunhtai** [comment]
hi @perclasson , I checked the material guide and couldn't find the mention of the cap (yet?) https://m3.material.io/components/top-app-bar/specs

do you know when the material spec will be public?

**Comment by perclasson** [comment]
I don't think those guidelines were ever published externally. We prevented the top bar text from scaling to 2x because it made it disproportionately large compared to the rest of the UI when the text size was increased.

IIRC, the normal text 2x would be the same as the app bar even when capped at 1.34x, so it should not limit someone with visual impairments to read the app bar text.

I am no longer working on Flutter or Material, so I cannot help out with this.

**Comment by chunhtai** [comment]
If this is internal requirement, there isn't anything we can do be. closing this as not planned.
